docs/topic-guides/system-concepts/scheduling.txt

```diff
@@ -119,36 +119,55 @@ enabled:
    priority 1 then starts running. Once that experiment is complete,
    distributed training experiment with priority 2 runs.
 
-*******************************
- Gang-scheduling on Kubernetes
-*******************************
-
-Kubernetes does not natively support gang-scheduling and its default
-scheduler will schedule pods on a first-come-first-serve basis. This
-approach is problematic if a user submits several multi-pod jobs at
-once; first-come-first-serve could result in a cluster deadlock.
-Determined is able to support gang-scheduling by using the lightweight
-coscheduling plugin, which extends the Kubernetes scheduler and blocks
-scheduling of pods unless there are enough resources for all the pods in
-the job. To function, the plugin requires special labels to be set that
-specify the amount of nodes that each job needs for execution.
-Determined is able to automatically calculate and set these labels for
-its experiments.
-
-Importantly, the coscheduling plugin does not work with cluster
-autoscaling. Static node pools must be used to achieve gang-scheduling.
-Also, while the plugin allocates resources to jobs based on their
-priority, it does not support preemption. Any low priority task will be
-able to finish before a higher priority task can begin running.
-Additionally, there isn't an implementation of ``max_slots`` or
-``max_concurrent_trials`` that would limit the resources of an
-experiment, i.e. one for hyperparameter search. Lastly, Determined's
-capability to automatically set pod labels is restricted to GPU
-experiments; it is unable to do the same for CPU experiments or user
-commands. If gang-scheduling is desired for these, it must be set
-manually via the environment field in the config. For instance:
-
-.. code::
+.. _scheduling-on-kubernetes:
+
+**************************
+ Scheduling on Kubernetes
+**************************
+
+By default, Kubernetes schedules pods one at a time, in a
+first-come-first-serve (FCFS) manner. This can result in problematic
```

Comment:
Blocking: see first comment about FCFS. Mea culpa - FCFS was a bad way to describe it to begin with.